# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gnome-python-extras/gdl-2.24.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./gdl/gdl.override.orig	2009-04-11 16:38:30.000000000 +0300
+++ ./gdl/gdl.override	2009-04-11 16:39:24.000000000 +0300
@@ -6,7 +6,6 @@
 #include <pygtk/pygtk.h>
 #include <gdl/gdl-dock.h>
 #include <gdl/libgdltypebuiltins.h>
-#include <gdl/gdl-icons.h>
 #ifndef HAVE_GDL_0_7
 # include <gdl/gdl-dock-bar.h>
 #endif
